memory/training/siamese/trainer.py

```diff
@@ -17,8 +15,9 @@
 
 from memory.training.utils import (FileTemplates, DirTemplates,
                                    ImageDataset, DataGenerator,
-                                   build_contrastive_loss, GeneralConstants)
-
+                                   build_contrastive_loss)
```

Comment:
Can you add this import to `memory/model/nearest_neighbors/__init__.py` and then import it from there? I generally try to to import directly from other files unless they are in the same directory level.

Reply:
Sorry, I'm a bit confused by this. Are you talking about the entire `from memory.training.utils import (FileTemplates, ... ,build_contrastive_loss)` import? I'm not sure why we would want to put that specifically in the `nearest_neighbors/__init__.py` script.